### DISCUSSION TITLE
Running command every x seconds

### DISCUSSION THREAD

**Discussion by jfbauer432** [StreamController/StreamController#210] [Q&A] [answered]
If you have an action on a page in StreamController that runs a command every x seconds.  Is it expected that this will run even when that page is not being displayed?  From what I am seeing, it won't run the command until you initially display that page.  but will then continue to do so even after switching to another page.

**Comment by Core447** [comment, marked as answer]
Yeah, that's not the intended behavior. Can you please open an issue on the [OS plugin](https://github.com/streamController/osplugin)?

**Comment by jfbauer432** [comment]
Created StreamController/OSPlugin#17
